File: term2/week1-rs/exercise6.rs
use std::io;

fn main() {

    let mut width = String::new();
    println!("Enter the width: ");
    io::stdin()
        .read_line(&mut width)
        .expect("Failed to read, get good");
    let width = width.trim().parse::<i32>().unwrap();

    println!("\n");

    for i in 0..width {
        for e in 0..(i+1) {
            print!("{} ", e+1)
        }
        println!();
    }
}

File: term2/week1/week1-c/exercise6.c
#include <stdio.h>

int main() 
{
    int number;

    printf("Enter the width: ");
    scanf("%i", &number);

    for (int i = 0; i < number; i++)
    {
        for (int x = 0; x < (i+1); x++)
        {
            printf("%i", (x+1));
        }
        printf("\n");
    }
    
    return 0;
}